README.md

```diff
@@ -15,6 +15,12 @@ The MarkLogic Connector for Hadoop is an extension to Hadoop’s MapReduce frame
 
 ## Release Note
 
+### What's New in mlcp and Hadoop Connector 11.0.3
+- Removed an unused dependency.
```

Comment:
What dependency?

Reply:
It was a json dependency that got removed... Let me change the line to be more informative.